src/main/pages/user-management/permissions.adoc

```diff
@@ -74,13 +74,41 @@ The following permissions are applicable to organizations:
 |===
 
 [id="super-priviliged-mode"]
-== Super Priviliged Mode
+== manageSystem permission
 
-The permission "manageSystem" can be extended to provide a super privileged mode that allows advanced actions to be performed on any resources managed by the system. A user with "manageSystem" permission is able read and stop any workspaces. To perform other actions on workspaces and organizations, the user will need to assign himself the permissions needed.
+User with `manageSystem` are able to perform various admin operations.
+On Start, Che will grant this permission to the admin user, with default name `admin`.
```

Comment:
On che-master start or after the first login, this permission will be granted to the person configured with ```CHE_SYSTEM_ADMIN__NAME``` property.